processor/transformprocessor/internal/metrics/func_convert_exponential_hist_to_explicit_hist.go

```diff
@@ -0,0 +1,305 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"golang.org/x/exp/rand"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottlmetric"
+)
+
+type convertExponentialHistToExplicitHistArguments struct {
+	DistributionFn string
+	ExplicitBounds []float64
+}
+
+// distributionFnMap - map of conversion functions
+var distributionFnMap = map[string]distAlgorithm{
+	"upper":    upperAlgorithm,
+	"midpoint": midpointAlgorithm,
+	"random":   randomAlgorithm,
+	"uniform":  uniformAlgorithm,
+}
+
+func newconvertExponentialHistToExplicitHistFactory() ottl.Factory[ottlmetric.TransformContext] {
+	return ottl.NewFactory("convert_exponential_hist_to_explicit_hist",
```

Comment:
lets use exact otel terms in the name: `convert_exponential_histogram_to_histogram`

Reply:
Function name updated to: `convert_exponential_histogram_to_explicit_histogram`

Reply:
The otel spec doesn't use the term `explicit` in the name as far as I know.  Using only `histogram` should be sufficient.

Reply:
Removed